1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,31 @@
 // Write your code here
+const catsPic = [
+  "cats/cat0.jpg",
+  "cats/cat1.jpg",
+  "cats/cat2.jpg",
+  "cats/cat3.jpg",
+];
```

Comment:
this is really good - I like how you're using an array to store the images & updating the img src based on the current slide